### PR TITLE
MM-54125 - Create Looker charts for Stage progression on Opportunities

### PR DESCRIPTION
**Pull request by sri-byte**
Impact: Add custom fields to Opportunity & Opportunity Snapshot for ticket https://mattermost.atlassian.net/browse/MM-54125.

Custom fields required for these charts are
- cpq_renewal_arr__c
- exit_year_arr__c
- sales_forecast_category__c
- total_net_new_arr_with_override__c

### CODE REVIEW

File: views/orgm/opportunity.view.lkml [hunk 1]
```diff
@@ -101,6 +108,34 @@ view: opportunity {
...
+  dimension: sales_forecast_category__c {
+    type: string
+    sql: ${TABLE}.sales_forecast_category__c ;;
+    description: "Sales Forecast Category"
+  }
+
+  dimension: total_net_new_arr_with_override__c {
+    type: number
+    sql: ${TABLE}.total_net_new_arr_with_override__c ;;
+    description: "Total Net New Arr with Override"
```
Comment (catalintomai) [inline views/orgm/opportunity.view.lkml:135]: nit: Arr-> ARR (to be consistent with the others)

Comment (catalintomai) [inline views/orgm/opportunity.view.lkml:135]: nit: Arr-> ARR (to be consistent with the others)

File: views/orgm/opportunity.view.lkml [hunk 2]
```diff
@@ -101,6 +108,34 @@ view: opportunity {
   # Dimensions
   #
 
+
+  dimension: cpq_renewal_arr__c {
```
Comment (catalintomai) [inline views/orgm/opportunity.view.lkml:112]: side note (for a future refactoring) - wondering why for dimensions we have `type`, `sql`, `description` ordering of fields and for measures we have a different `description`,`label`,`sql`,`type` ordering.

Comment (catalintomai) [inline views/orgm/opportunity.view.lkml:112]: side note (for a future refactoring) - wondering why for dimensions we have `type`, `sql`, `description` ordering of fields and for measures we have a different `description`,`label`,`sql`,`type` ordering.

Comment (sri-byte) [reply views/orgm/opportunity.view.lkml:112]: Ordering isn't important, but we could do a revisit later and make it consistent.

**Review by catalintomai — APPROVED** [review]

File: views/orgm/opportunity.view.lkml (hunk 2 above)
Comment (catalintomai) [reply views/orgm/opportunity.view.lkml:112]: yes, it's not consequential, but if we maintain a consistent ordering it is easier to spot if smth. is missing than if things are different from one entity to another.

**Pull request merged**